tests/nat/eval/runners/red_teaming_runner/test_red_teaming_config.py

```
@@ -0,0 +1,290 @@
+
+    def test_raw_scenario_with_invalid_evaluator_dict_fails(self):
+        """_RedTeamingScenarioRaw with invalid evaluator dict should fail validation."""
+        scenario_raw = _RedTeamingScenarioRaw(
+            middleware=RedTeamingMiddlewareConfig(attack_payload="test"),
+            evaluator={
+                # Missing required fields: llm_name, judge_llm_prompt, filter_conditions
+                "reduction_strategy": "last",
+            },
+        )
+
+        with pytest.raises(Exception):  # Pydantic ValidationError
```

Ruff (0.14.8)

Do not assert blind exception: `Exception`

(B017)